src/components/profiles/members-list.vue

```diff
@@ -28,10 +30,10 @@ export default {
 </script>
 
 <template lang="pug">
-.members-list(ref="scrollContainer")
+.members-list(ref="$refs.scrollContainer")
```

Comment:
> 

@pugakn I think this is true because I have seen the same use of $refs in other parts of the application, and it also works. Just scrollContainer doesn't work. This was the bug.

Reply:
> https://vuejs.org/guide/essentials/template-refs.html#refs-inside-v-for when assigning a name to `ref` it is added automatically under this.$refs, you don't need to add `$refs.` to the name @evgenibir

Fixed!